content/docs/user-guide/data-pipelines/reproducing-pipelines.md

```diff
@@ -0,0 +1,83 @@
+stage if its dependencies are invalid.
+
+```dvc
+$ dvc repro
+'data.dvc' didn't change, skipping
+Stage 'train' didn't change, skipping
+Data and pipelines are up to date.
+```
+
+You can force to run a pipeline if its dependencies are valid with `--force`
+flag. `dvc repro` runs the stages in topological order. It runs dependencies
+before dependents. It doesn't run the stages in parallel if they are separate in
+the pipeline DAG.
+
+It shows the command output in the same terminal the command is issued. If you
+want to redirect the command output, you can modify the commands in `dvc.yaml`.
+
+DVC runs the `dvc.yaml` in the current directory by default.  
+If the directory you are in doesn't contain a `dvc.yaml` file, you can specify
+the directory with `--cd` option to `dvc`.
+
+```
+$ dvc --cd $HOME/myproject repro
+```
+
+You can have multiple pipelines distributed to multiple `dvc.yaml` files in
+different directories. You can run all of them at once with `--all-pipelines`
+flag. This works even if you're in a child directory in the project.
+
+```dvc
+$ dvc repro --all-pipelines
+```
+
+If you want to run the pipelines only in the child directories, you can also use
+`--recursive` flag. Instead of all pipelines in the project, it runs the
+pipeline in the current directory and its descendants.
+
+<admon type="warn">
+Note that DVC doesn't try to sort or find dependency relationship of multiple pipelines. 
+</admon>
+
+## Outputs
+
+A pipeline is built to create outputs like models, intermediate data, plots, or
+metrics. These outputs define the connection between stages and if a stage's
+dependencies are newer than its outputs, the stage is considered invalid.
+
+Outputs are produced using stage commands. DVC doesn't make any assumption
+regarding how these outputs are produced so you can use any shell command to
+produce outputs.
+
+From more general to more specific, the types of outputs are as follows:
+
+- **Files and Directories**: Both data to feed to intermediate stages, and
+  models that are produced as final outputs are within this type. DVC tracks the
+  changes in these files but doesn't make any assumptions regarding their use.
+
+- **Metrics**: DVC pipelines has special support for metrics files. These files
+  contain model comparison metrics that are output after training, validation or
+  testing phases of model production. DVC allows to compare produced metrics
+  with one another using `dvc metrics diff` and presents the results as a table
+  with `dvc metrics show`.
+
+- **Plots**: A more specialized kind of metrics files can be used to create plot
+  graphs. In contrast to metrics files, plots are used to present progression of
+  more than one training step, like epochs in deep learning projects. DVC
+  supports to supply plots files to Vega-Lite and present them on browser with
+  `dvc plots show`.
+
+- **Image Plots**: Another kind of output is plots produced by the commands.
```

Comment:
let's make it part of plots

Reply:
and we are deprecating I think stage level plots overall (probably metrics also)  ... so may be the whole section is not that needed